app/frontend/src/submissions/views/SubmissionsHome.vue

```diff
@@ -199,55 +209,61 @@ export default {
         owner_mailing_address: '',
         owner_city: '',
         owner_province_state: '',
-        owner_postal_code: ''
+        owner_postal_code: '',
+
+        // non-form fields that should be saved with form
+        meta: {
+          drillerSameAsPersonResponsible: false
+        }
       }
     },
     saveForm () {
       // saves a copy of form data locally
-      this.saveFormReset()
+      this.saveStatusReset()
       const data = JSON.stringify(this.form)
       localStorage.setItem('savedFormData', data)
       setTimeout(() => { this.saveFormSuccess = true }, 10)
       setTimeout(() => { this.saveFormSuccess = false }, 1000)
```

Comment:
Is this test/debug code?

Reply:
it's not for debugging, it automatically removes the "save success" indicator (click save, checkbox appears after save is successful, checkbox disappears after 1 second)